randovania/data/json_data/prime1.txt

```diff
@@ -1718,6 +1775,7 @@ Asset id: 2758909034
           Missile ≥ 2 and Grapple Beam and Heat-Resisting Suit
           Missile ≥ 2 and Space Jump Boots and Heated Rooms Damage ≥ 161
           Space Jump Boots and Heated Rooms Damage ≥ 145 and Shoot Super Missile
+          Missile ≥ 2 and Morph Ball Bomb and Morph Ball and Heat-Resisting Suit
```

Comment:
Not sure I understand what Morph/Bombs are for here, also the suit. Can be grabbed with a slope jump nsj.

Reply:
You can do a set of single bomb jumps to scale it from one side NSJ, as an alternative to said slope jump. I will however have to catalogue the slope jump later.

Reply:
Should add in the bomb jump trick then if keeping it.